app/javascript/client/checkout/Checkout.tsx

```diff
@@ -101,18 +250,10 @@ const InternalCheckout: FC<CheckoutProps> = ({ history, unitPrice, cart, subtota
 			attn: useRef(),
 		} as { [key: string]: React.RefObject<HTMLInputElement> };
 
-    if (!productData) return null;
```

Comment:
I just moved this above the `handleSubmit` function because I thought I needed something from it in `handleSubmit` (turns out I didn't actually but whatever)